src/pages/ServiceGraph/SummaryPanelEdge.tsx

```diff
@@ -118,20 +115,13 @@ export default class SummaryPanelEdge extends React.Component<SummaryPanelPropTy
   }
 
   private updateCharts = (props: SummaryPanelPropType) => {
+    const filters = ['request_count', 'request_duration', 'request_error_count'];
+    const byLabelsIn = ['source_service', 'source_version'];
```

Comment:
I do not know what these are used for, however, if this is somehow related to the Prometheus labels (they look like them at least), there is no such thing as "source_service" label anymore.

Reply:
Thanks for the heads up. I have no idea either but is sent to `api/namespace/{namespace}/service/{service}`. I'll check what is that used for.